tools/bazel.sh

```diff
@@ -18,5 +18,28 @@ if [ ! -d "${INSTALL_BASE}" ]; then
   done
 fi
 
-# Finally run Bazel with the new intall base
-"${BASEDIR}/bin/bazel-real" --install_base="${INSTALL_BASE}" "$@"
+export BAZEL_REAL="${BASEDIR}/bin/bazel-real"
```

Comment:
This variable is unused?

Reply:
It's used externally, but I've also added an internal use